paymentrails.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.license = "MIT"
   s.author = "PaymentRails"
   s.files = Dir.glob ["README.rdoc", "LICENSE", "lib/**/*.{rb,crt}", "spec/**/*", "*.gemspec"]
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.7'
```

Comment:
2.4 is end of life. Even 2.7 is nearing EOL next April

https://www.ruby-lang.org/en/downloads/branches/

it'd be good to stay on stable versions

Reply:
@zero-bites Which version is stable now, and does it introduce too many breaking changes?

Reply:
currently versions above 3.0 are stable, however it's not 100% backwards compatible with 2.7 because of a slight change in method signatures

I recommend completing testing before doing a major upgrade

On the other hand ruby 3.0 and beyond supports strict typing with RBS: https://github.com/ruby/rbs

Which could be useful for parsing API responses